event-handlers/src/test/java/no/sikt/nva/nvi/events/evaluator/EvaluateNviCandidateHandlerTest.java

```diff
@@ -165,7 +167,8 @@ void shouldCreateNewCandidateWithPointsOnlyForNviInstitutions() throws IOExcepti
         var messageBody = getMessageBody();
         var institutionPoints = ((NviCandidate) messageBody.candidate()).institutionPoints();
         assertEquals(1, institutionPoints.size());
-        assertTrue(institutionPoints.containsKey(CRISTIN_NVI_ORG_TOP_LEVEL_ID));
+        assertTrue(institutionPoints.stream().map(
+            InstitutionPoints::institutionId).anyMatch(uri -> uri.equals(CRISTIN_NVI_ORG_TOP_LEVEL_ID)));
```

Comment:
Is it an object access pattern to filter by URI for InstitutionPoints? If so, probably create a method for this.